### DISCUSSION TITLE
Pooling working?

### DISCUSSION THREAD

**Discussion by Yamanipanuchi** [guydavis/machinaris#169] [Q&A] [answered]
I added my pools using the GUI on a windows machine. Using the same key on each system. How do I know that pooling it working on the docker? 

I've run the following,

root@Tower:/chia-blockchain# chia plotnft show
Wallet height: 557719
Sync status: Synced
root@Tower:/chia-blockchain#

But nothing really tells me that the Pooling part is there and working.

**Comment by nomadsgalaxy** [comment]
in the docker, please run;
chia plotnft show

you should see something like;
```
Wallet id 3: 
Current state: FARMING_TO_POOL
```

**Reply by Yamanipanuchi** [reply]
Ya above I did that. It does not say FARMING_TO_POOL. So Something isnt right.

**Reply by Yamanipanuchi** [reply]
I have one plot that was created with a POOL NFT, But I did not create the plot with the docker image. I did it on a separate machine and then transferred it over.

From the looks of it, My docker image is not aware of any POOL NFT's.

**Reply by nomadsgalaxy** [reply]
Please provide the full output of;
`chia plotnft show`

**Reply by nomadsgalaxy** [reply]
Also, please keep posts to this thread to avoid "Suggested Answer" Spam.

**Reply by guydavis** [reply]
@Yamanipanuchi Please provide the full output of `chia plotnft show` as you were asked above.  

Also, please provide full output of both your `chia wallet show` and `chia keys show`.  (redact if you need to)  

How many keys did import into Machinaris?  Indicate whether each key import happened via the Machinaris WebUI Setup screen or by the in-container CLI with a `chia keys add` command you typed.

Finally attach [all logs](https://github.com/guydavis/machinaris/wiki/Troubleshooting#nothing-works--where-are-the-logs) for us to review to this Discussion thread.

Please help us, help you.  Thanks much.

**Comment by Yamanipanuchi** [comment]
All the GUI version of Chia picked up the POOLNFT's no issue. This docker unfortunatly is the only one that seems to not work. This docker happens to be my primary Chia farmer.

**Comment by Yamanipanuchi** [comment]
Just mapped the drives to a windows machine and added them to chia and Pool plots look correct.

![image](https://user-images.githubusercontent.com/4724631/125207567-f076c800-e241-11eb-8b15-2b7b180ebe9b.png)

**Comment by Yamanipanuchi** [comment]
Unless I completely missed something, The docker is not picking up existing PoolNFT's. Might need to turn this into an issue, I looked on Chai's docker github, It hasn't been updated for 20 some days.. So.

**Reply by nomadsgalaxy** [reply]
Just to confirm, are are you using Machinaris via a Docker?

**Reply by Yamanipanuchi** [reply]
Yes, on an unRAID system.

**Reply by nomadsgalaxy** [reply]
How did you install Machinaris to Unraid? Was it via Docker or via the app install?

**Reply by Yamanipanuchi** [reply]
Via app install.

**Reply by Yamanipanuchi** [reply]
Version does show 1.2.1 dev

**Comment by Yamanipanuchi** [comment]
Didnt notice this, The `chia poolnft show` is in the pool menu. Still doesnt show that its mining for a pool.

![image](https://user-images.githubusercontent.com/4724631/125293809-9f65e300-e2d8-11eb-830e-178316a44fd0.png)

**Reply by Yamanipanuchi** [reply]
The plot that is attached to the poolnft I sleected is in the list of plots on the system.

![image](https://user-images.githubusercontent.com/4724631/125294969-99bccd00-e2d9-11eb-8451-9aca2ac2fc55.png)

**Reply by Yamanipanuchi** [reply]
Its the only K34 plot I have right now.

**Comment by Yamanipanuchi** [comment]
Could this possibly be something to do with that fact that I have old plots that are not attached to a pool?

**Comment by Yamanipanuchi** [comment, marked as answer]
So issue now resolved. I dont completely understand why, but I deleted the wallet DP and let it re-sync and now the pools show up correctly.

*shrug*
